connector/src/test/scala/com/microsoft/kusto/spark/WriterTests.scala

```diff
@@ -142,8 +139,8 @@ class WriterTests extends FlatSpec with Matchers {
         "asd2" -> Row(Array("stringVal2\b\f")))
     )
 
-    // 2018-06-18 16:04:00.0 in GMT
-    val ts = new java.sql.Timestamp(1529337840000l)
+    val tz  = ZonedDateTime.parse("2018-06-18T15:00:00.123456789+04:00")
```

Comment:
I could be missing it, but I don't see an assert that this or any non-UTC LocalDateTime went through the writer and came out the other end as expected.

Reply:
line 282
it sohuld equal to 
sdf.format(ts.toInstant.atZone(tz.getZone))